airflow/www/static/css/bootstrap-theme.css

```diff
@@ -3252,7 +3252,8 @@ tbody.collapse.in {
   content: "\e223";
 }
 .glyphicon-duplicate:before {
-  content: "\e224";
+  font-family: "FontAwesome";
```

Comment:
This is an unrelated change. Can you revert it?  We are working on full font-awesome migration: https://github.com/apache/airflow/pull/6254

Reply:
Sure, I wasn't aware this was taken care of already in a different PR